src/Kunstmaan/SitemapBundle/Model/SitemapIndex.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Kunstmaan\SitemapBundle\Model;
+
+class SitemapIndex
+{
+    private $url;
+
+    public function __construct(string $url)
+    {
+        $this->url = $url;
+    }
+
+    public function getUrl(): string
+    {
+        return $this->url;
+    }
+}
```

Comment:
Can you also implement the other properties defined in the spec? (lastmod, priority, etc) See https://www.sitemaps.org/protocol.html

So we are fully feature complete!

Reply:
@acrobat Only the lastmod (optional) property is available for the sitemap index. I can add this but this is also not added in the current file. Do you want to add some value for the current lines? Not sure where you want to configure this value then.